### DISCUSSION TITLE
July 30 - Literate Programming VI - personal style

### DISCUSSION THREAD

**Discussion by tonyfast** [Quansight/writers-workshop#26] [New class]
# the goal of the workshop

[your style](https://gist.github.com/tonyfast/359ed1f7d2df21d9e79c0f996f795d57)

## so far

1. introduction to literate programs where code and narrative contribute equally to the narrative.
2. introduction to different media formats in the notebook and their use as collage.
3. introduction to testing a mess of notebooks.
4. restart and run all or it didn't happen
5. your brand

## participating on jitsi

* use the jit.si meet only https://meet.jit.si/litprog-writers-workshopbounce-house
* join from the [binder](https://gke2.mybinder.org/v2/gh/deathbeds/hourhaus/writers-workshop-7-29-2021?urlpath=git-pull%3Frepo%3Dhttps%253A%252F%252Fgist.github.com%252Ftonyfast%252F359ed1f7d2df21d9e79c0f996f795d57%26urlpath%3Dlab%252Ftree%252F359ed1f7d2df21d9e79c0f996f795d57%252Fstyle.ipynb%26branch%3Dmaster)
* you finally have to option to join the interactive binder, this not available until class starts.

## teacher's preferred ground rules

* How to use the raise handle feature? Keeping the professor on track..
* I encourage y'all to interrupt me when I'm talking or demo'ing. 
  Sometimes I get distracted by my own special interests and this class if for you all.
* To help take notes, please respond to this discussion item and continue editing your one reply during class. i can see the changes on the discussion board as they happen.

## [calls to action] :+1: :-1:  :smile: :confused:  :rocket:  :heart:  :eyes: :tada: attendance

* place any questions you have in the discussions and we can try to discuss them during the workshop.

* react to this discussion so we can take attendance.

![image](https://user-images.githubusercontent.com/4236275/117461813-fd163180-af1b-11eb-901b-bfdee65d8bdc.png)

### Q1 how's your energy

* :+1: i feel energetic from this week
* :-1: i feel exhausted from this week

## note taking

please use this discussion as a place to take notes. create a comment that you edit throughout the lesson. 

_the perfect is the enemy of the good_

## agenda

0. [:00-:07] chit chat
1. show and tell
2. evolving style
4. parting words
5. [:50-:60] wrapup

write! right?

**Comment by isabela-pf** [comment]
Notes!

Show and tell: 
- Mars shared the alt text task force exploratory graphics and branding 🐢 
- Matthew shared the work in progress document about his [internship project](https://nbviewer.jupyter.org/gist/honno/bf7bd9ecbf6926cb68a193a068a55033)
- Bruno shared work-in-progress photos from a recent work trip

Evolving style:
- Returning to old projects or things that you didn't understand in the past and understanding it this time around is a sign of changing literacy
- What programming language were you writing ten years ago?
- "Pace yourself. It's a long career."
- Tangent: https://en.wikipedia.org/wiki/Hysteresis
- When you write a lot, searching and storing it all can be a problem. Be perpared.
- Keep track of yourself and your work on a month-level time span, not days (too short)

Influential writing
Except I only caught some of the people.
- [Jane Jacobs](https://en.wikipedia.org/wiki/Jane_Jacobs)
- [Richard Hamming](https://en.wikipedia.org/wiki/Richard_Hamming)
- [Walter Gropius](https://en.wikipedia.org/wiki/Walter_Gropius)
- [Roland Barthes](https://en.wikipedia.org/wiki/Roland_Barthes)
- [Monoskop index](https://monoskop.org/Index)

**Comment by MarsBarLee** [comment]
Notes:
It takes two to tango and then find your personal style
Tony encourages to make a zine or coding band
Conversation between libraries
We're creating a code development economy- barter with lines of code?
greatness is a matter of style - hamming
make things, a blog, essay, share with the community- a la writer's workshop!
